models/Document/Link.php

```diff
@@ -268,7 +268,7 @@ public function setInternalType(?string $type): static
         return $this;
     }
 
-    public function getElement(): ?Model\Element\ElementInterface
+    public function getElement(): Model\Element\ElementInterface|Model\Element\ElementDescriptor|null
```

Comment:
I think getElement() shouldn't return a ElementDescriptor. When the object attribute is a ElementDescriptor it should be reloaded.